pkg/telemetry/prometheus/quality.go

```diff
@@ -0,0 +1,47 @@
+package prometheus
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/livekit/protocol/livekit"
+)
+
+var (
+	qualityRating prometheus.Histogram
+	qualityScore  prometheus.Histogram
```

Comment:
This is the psuedo-MOS score. So, can do p50, p90, p95. Also has more granular bins.